tests/e2e/test_cli.py

```diff
@@ -140,6 +141,82 @@ def test_pyproject_common_statements(runner: CliRunner, tmpdir: LocalPath) -> No
     assert test_file.read() == PYPROJECT_CONFIG_FIXED_SOURCE
 
 
+@pytest.mark.parametrize(
+    ("create_global_conf", "use_local_conf", "create_pyproject", "expected_imports"),
+    [
+        pytest.param(True, False, False, "from g import G", id="global"),
+        pytest.param(False, True, False, "from l import L", id="local"),
+        pytest.param(False, False, True, "from p import P", id="pyproject"),
+        pytest.param(
+            True, True, False, "from g import G\nfrom l import L", id="global-and-local"
+        ),
+        pytest.param(
+            True,
+            False,
+            True,
+            "from g import G\nfrom p import P",
+            id="global-and-pyproject",
+        ),
+        pytest.param(False, True, True, "from l import L", id="local-and-pyproject"),
```

Comment:
Shouldn't this be `from l import L\nfrom p import P`?

Reply:
Updated in https://github.com/lyz-code/autoimport/pull/206/commits/f22c4a13dc71d79aee7cc67eaa97007108279028.

Reply:
Lovely, thanks!